front/tests/pages/map-model.ts

```diff
@@ -47,12 +56,12 @@ class PlaywrightMap {
     await this.getSearchedStation.fill(station);
   }
 
-  async clickOnMap(position: { x: number; y: number }) {
-    await this.getMap.click({ position });
+  async clickOnMap() {
```

Comment:
So here we click on the center of the map right ? Can you add a comment to precise it
```suggestion
  /** click on the center of the map */
  async clickOnMap() {
```